recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.0.2" %}
+
+package:
+    name: pyamg
+    version: {{ version }}
+
+source:
+    fn: pyamg-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pyamg/pyamg-{{ version }}.tar.gz
+    md5: bd024253ebae6f7bf2d89fe32fc5d64a
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy x.x
+        - zlib  # [linux]
+    run:
+        - python
+        - numpy x.x
+        - scipy
+        - zlib  # [linux]
+
+test:
+    requires:
+        - nose
+    imports:
+        - pyamg
```

Comment:
@jakevdp in that case do whatever is easier for you.  When the test is just 1-line I add it in the `meta.yaml`. When it is more than 1 line I go for a separated script.